templates/aws/aws_cloudformation_template.yml

```diff
@@ -5,16 +5,28 @@ Description:
   CloudFormation template for LANSA Database regression Testing.
 
 Parameters:
-  LANSA_VERSION:
+  LANSAVERSION:
     Type: String
     Description: LANSA Version that needs to be test
-  AMI_ID:
+  AMIID:
     Type: String
-    Description: AMI_ID for provisioning the VM    
-  MYSQL_SNAPSHOT_ARN:
+    Description: AMI_ID for provisioning the VM
+  SECURITYGROUPID:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_SG_ID
+    Description: Security Group to be attached to the VM.
+  SUBNETIDFORVM:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_SUBNET_ID
+    Description: Subnet ID where VM would be provisioned.
```

Comment:
Just use the default subnet

Reply:
Subnet id is fetched from parameter store and its value is one of the subnets in Default VPC.